lib/stm32/common/i2c_common_v1.c

```diff
@@ -463,4 +463,26 @@ void i2c_clear_dma_last_transfer(uint32_t i2c)
 	I2C_CR2(i2c) &= ~I2C_CR2_LAST;
 }
 
+void i2c_set_speed(uint32_t p, enum i2c_speeds speed, uint32_t clock_megahz)
+{
+	i2c_set_clock_frequency(p, clock_megahz);
```

Comment:
The first param name looks confusing here. It makes sense to follow current de-facto convention and call it ```i2c```.